include/cscore_c.h

```diff
@@ -1,16 +1,17 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) FIRST 2016. All Rights Reserved.                             */
+/* Copyright (c) 2016-2017 FIRST. All Rights Reserved.                        */
 /* Open Source Software - may be modified and shared by FRC teams. The code   */
 /* must be accompanied by the FIRST BSD license file in the root directory of */
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#ifndef CSCORE_C_H_
-#define CSCORE_C_H_
+#ifndef INCLUDE_CSCORE_C_H_
+#define INCLUDE_CSCORE_C_H_
 
-#include <stddef.h>
 #include <stdint.h>
 
+#include <cstddef>
```

Comment:
No.  The only C headers are cscore.h and cscore_c.h.  This is the case for ntcore as well.